planning/testnet-kpis.md

```diff
@@ -1,29 +1,40 @@
-| Wallet extension downloads| Good proxy for the number of active users. Straightforward to capture.| GitHub|Number of wallet extension downloads in the last 4 weeks||
+| Testnet uptime | Captures how robust and ready for mainnet Obscuro is | DataDog avg:system.uptime{*} | Average Testnet uptime over the last 4 weeks|99%|
+| Number of wallets connected to Obscuro Gateway| Good proxy for the number of active users. Straightforward to capture.| Datadog? |Number of daily connections|500|
+| Number of transactions| Typical guage of the amount of activity on testnet (even though it can be gamed).| Datadog? |Number of daily transactions|2000|
+
+### Selected KPIs for Developers
+| KPI NAME | RATIONALE | SOURCE | METRIC | TARGET |
+|--|--|--|--|--|
+| Testnet documentation page views (https://docs.obscu.ro/testnet/) | Gauges interest from developers in building on testnet. A good early indicator. Straightforward to capture | Google Analytics | Number of unique page views in the last 4 weeks |50|
+|DApps in development| This is difficult to measure as it’s not something we can look to the network to accurately measure. We can use our business development CRM data as an indicator. | Airtable "Partnerships" tab| Number of partners with status changed to "Soft Commitment" in the last 4 weeks|40|
+| New dApps deployed on Testnet| Indicates how much new activity and commitment of effort from dApp builders| ObscuroScan?| Number of new dApp addresses deployed in the last 4 weeks|20|
 
 ## Discounted KPIs
-These KPIs were considered and discounted.
+These KPIs were considered and discounted for now.
 
+- Wallet extension downloads
+    - Good proxy for the number of active users.
+    - Straightforward to capture.
+    - Discounted because Obscuro Gateway is the recommended wallet connection method.
 - Ported Solidity dApps– original vs fork
     - One of Obscuro’s key promises is that existing Solidity dApps will just work on Obscuro. This KPI helps track whether Obscuro is fulfilling this promise. The reason for differentiating between original fork e.g. Uniswap vs PancakeSwap is it further provides insight into whether original teams believe in Obscuro.
     - Discounted because very difficult to determine of the dApps on Testnet which have been ported.
-- Number of transactions
-    - Gauge the amount of activity on Testnet.
-    - Discounted because it can be gamed too easily leading to misleading results.
 - Number of nodes
     - Nodes secure the network and track general interest in Obscuro. This KPI also provides an input into the staking function and has an impact on tokenomics.
-    - Discounted for now because too early in the lifecycle of Testnet to bring nodes online.
+    - Discounted for now because Obscuro Labs will run the nodes. When validator nodes can be run by others, this KPI will be introduced.
```

Comment:
I wonder if we should track the number of applications to become a node operator at this stage through another typeform form

Reply:
We already have a list of people who want to operate a node - are you saying it could be stale?

Reply:
Yes, it's coming up to 1.4 years now since we captured